layouts/shortcodes/pentest-report-requirements.html

```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you generally must set up a test of at least eight (8) credits. 
+If you want a pentest report, you generally must set up a test of at least five (5) credits. 
```

Comment:
Platform does not support less than 5 credit comprehensive PT so not sure we need to mentioned this. In the past we supported smaller compliance PTs. Also, I think we should refer to all current PT report as Full Report Vs when we start adding content about Agile PT then we will refer to those reports as Automated Report.

Reply:
Removed.